lib/src/Pos/Lrc/DB/Leaders.hs

```diff
@@ -44,13 +62,49 @@ prepareLrcLeaders ::
        , HasConfiguration
        )
     => m ()
-prepareLrcLeaders =
-    whenNothingM_ (getLeaders 0) $
-        putLeaders 0 genesisLeaders
+prepareLrcLeaders = do
+    whenNothingM_ (getLeadersForEpoch 0) $
```

Comment:
TBH, I would rework LRC DB initialization the same way as it's done for GState DB (i. e. have initialization flag), but it can be done later.